test/sample_test.go

```diff
@@ -38,57 +40,74 @@ var _ = Describe("Generating options", func() {
 			OptionMyFloat(4.56),
 			OptionMyString("my-string"),
 		)
-		Ω(err).NotTo(HaveOccurred())
-		Ω(cfg.myInt).To(Equal(123))
-		Ω(cfg.myFloat).To(Equal(4.56))
-		Ω(cfg.myString).To(Equal("my-string"))
+		Ω(err).ShouldNot(HaveOccurred())
```

Comment:
Just some cleanup of ginkgo matchers: Ω and Should should pair up (as opposed to the alternate pairing of Expect and To).

Reply:
one of the ways ginkgo api isn't super tight.  but, still a fan of course.